src/ApplicationCore/Entities/FraudProtectionApiModels/SignInHistory.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Contoso.FraudProtection.ApplicationCore.Entities.FraudProtectionApiModels
+{
+    public class SignInHistory
```

Comment:
At least in this sample project we typically put all the sub/nested classes in the file with the top level request.  So, can you move this class definition into the SignIn class file?